src/constants/routes.js

```diff
@@ -1,21 +1,24 @@
 module.exports = {
   contactUs: '/contact',
-  accountInfo: '/dashboard/accountInfo',
-  paymentInfo: '/dashboard/paymentInfo',
-  supportCenter: '/dashboard/supportCenter',
+  accountInfo: '/dashboard/account-info',
```

Comment:
why the move to hyphenated?

Reply:
So it actually was hyphenated before the stubbed out dashboard routes..like the swag profile route..I don't care either way though. Which would you prefer?